channel/src/main/java/com/synopsys/integration/alert/channel/jira/server/JiraServerChannel.java

```diff
@@ -55,8 +57,9 @@ public JiraServerChannel(Gson gson, JiraServerChannelKey descriptorKey, AuditAcc
 
     @Override
     protected IssueTrackerContext getIssueTrackerContext(DistributionEvent event) {
-        FieldUtility fieldUtility = event.getFieldUtility();
-        return jiraServerContextBuilder.build(fieldUtility);
+        ConfigurationModel globalConfig = event.getChannelGlobalConfig()
+                                              .orElseThrow(() -> new AlertRuntimeException(new AlertConfigurationException("Missing Jira Server global configuration")));
```

Comment:
Why do we have the 2 exceptions here, rather than just the AlertRuntimeException?

Reply:
I didn't want to modify the method signature, but a missing global configuration is an `AlertConfigurationException`. It will be wrapped, but will make it more clear what the problem is.

Reply:
Perhaps we can introduce a new exception in a separate PR?

Reply:
Since we do this in a few places, we should try to figure out why this exception can be thrown in a place that does not allow checked exceptions before we assume that a new exception is needed.

Reply:
Jira issue created.

Reply:
We change the method signature in the future to throw an AlertConfigurationException since that appears to be the correct action.  That should be done in a smaller PR.